examples/app-npm-webpack/app/index.js

```diff
@@ -0,0 +1,53 @@
+var currencyFormatter, dateFormatter, numberFormatter, relativeTimeFormatter, startTime,
+	Globalize = require( "globalize" );
+
+startTime = new Date();
+
+currencyFormatter = Globalize.currencyFormatter( "USD" );
+dateFormatter = Globalize.dateFormatter({ datetime: "medium" });
+numberFormatter = Globalize.numberFormatter({ maximumFractionDigits: 2 });
+relativeTimeFormatter = Globalize.relativeTimeFormatter( "second" );
+
+document.getElementById( "intro-1" ).innerHTML = Globalize.formatMessage( "intro-1" );
+
+// Standalone table.
+document.getElementById( "currency-label" ).innerHTML = Globalize.formatMessage( "currency-label" );
+document.getElementById( "date-label" ).innerHTML = Globalize.formatMessage( "date-label" );
+document.getElementById( "number-label" ).innerHTML = Globalize.formatMessage( "number-label" );
+document.getElementById( "relative-time-label" ).innerHTML = Globalize.formatMessage( "relative-time-label" );
+
+document.getElementById( "currency" ).innerHTML = currencyFormatter( 69900 );
+document.getElementById( "date" ).innerHTML = dateFormatter( new Date() );
+document.getElementById( "number" ).innerHTML = numberFormatter( 12345.6789 );
+document.getElementById( "relative-time" ).innerHTML = relativeTimeFormatter( 0 );
+
+// Messages.
+document.getElementById( "message-1" ).innerHTML = Globalize.formatMessage( "message-1", {
+	number: currencyFormatter( 69900 ),
+	currency: dateFormatter( new Date() ),
+	date: numberFormatter( 12345.6789 ),
+	relativeTime: relativeTimeFormatter( 0 )
+});
+
+document.getElementById( "message-2" ).innerHTML = Globalize.formatMessage( "message-2", {
+	count: 3
```

Comment:
Maybe for the demo purposes also increment the `count` from `0` to some number and back to `0`? This would demonstrate how plurals work with different values.